common/templates/emails/sync_learner_profile_data_email_change_subject.txt

```diff
@@ -0,0 +1,6 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers %>
+
+${_("Your {platform_name} account email has been updated").format(
+    platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
```

Comment:
@UmanShahzad Or perhaps `configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)` could happen outside of the e-mail templates and just get passed in as `platform_name`? Not sure if there is an edX-specific convention around this, but I did find a few templates in the repo that get `platform_name` from the context that is passed to them.

Reply:
I saw other email templates that don't mind doing the lookup twice, and keeps us from having to define more variables. If you're okay with it, I might just leave it as is, to keep more code changes to a minimum. It doesn't really affect anything as you can ascertain.

Reply:
@UmanShahzad OK, that means there's no edX-specific convention around this :) I'm fine with leaving this as is.

Reply:
:+1: